docs/migration-guides/external-providers-primitives-v2.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Migrate provider interfaces from backend.run to primitives
+description: Migrate to the primitives interface in external providers
+
+---
+
+# Migrate provider interfaces from backend.run to primitives
+
+## Motivation and context
+
+The evolution of quantum algorithms and scaling of quantum processing units (QPUs) motivated the
+introduction of the Qiskit [primitives interface](/guides/primitives) to provide a higher-level abstraction for
+accessing QPUs. This interface is optimized for two core tasks in quantum algorithm development: expectation value estimation (Estimator) and
+circuit sampling (Sampler).
+
+After a time of co-existence of the `backend.run` model and primitives model, where the primitives
```

Comment:
```suggestion
After a time of co-existence of the `backend.run` model and primitives model, when the primitives
```
(If I understand correctly, once the primitives evolved to V2, backend.run was deprecated - is that what this is saying?)